segmenters/nlp/SpacySentencizer/__init__.py

```diff
@@ -0,0 +1,96 @@
+from typing import Dict, List
+
+from jina.executors.segmenters import BaseSegmenter
+
+
+class SpacySentencizer(BaseSegmenter):
+    """
+    :class:`SpacySentencizer` split the text on the doc-level
+    into sentences on the chunk-level with spaCy as the backend.
+    The splitting technique is divided into two different approaches:
+    1) Default Segmenter: Utilizes dependency parsing to determine the rule for
+        splitting the text. For more info please refer to https://spacy.io/api/sentencizer.
+    2) Machine Learning-based Segmenter: Utilizes SentenceRecognizer model trained
+        on a dedicated data created for sentence segmentation. For more info please
+        refer to https://spacy.io/api/sentencerecognizer.
+
+    :param lang: pre-trained spaCy language pipeline
+    :param use_default_segmenter: boolean to determine which segmentation algorithm to use,
```

Comment:
It is weird to me to have a `boolean` and be described as a selection of `which` algorithm. There should be only 2 options right?

Reply:
yes, would it be better to describe it as follows?
```if True will use dependency based sentence segmentation, otherwise ml-based implementation will be chosen```

Reply:
Yes

Reply:
ok thanks. fixed